config/ldes-delta-pusher/handle-types-util.ts

```diff
@@ -0,0 +1,65 @@
+import { Changeset } from "../types";
+import { query } from "mu";
+import { publish } from "./publisher";
+import { log } from "./logger";
+
+export type InterestingSubject = {
+  uri: string;
+  ldesType: string;
+};
+
+type SubjectsToInterestingSubjects = {
```

Comment:
to me this type's name reads like it's a map, but it's actually a filter. function Maybe call it SubjectFilter?